scipy/stats/tests/test_distributions.py

```diff
@@ -6351,6 +6351,25 @@ def test_entropy(self):
         ref = stats.weibull_min.entropy(c) - np.log(0.5)
         assert_allclose(res, ref, rtol=1e-15)
 
+    # reference values were computed with mpmath
+    # from mpmath import mp
+    # mp.dps = 50
+    # def sf_mpmath(x, c):
+    #     x = mp.mpf(x)
+    #     c = mp.mpf(c)
+    #     if x > 0:
+    #          return mp.mpf(0.5) * mp.exp(-x**c)
+    # else:
+    #          return mp.one - mp.mpf(0.5) * mp.exp(-(-x)**c)
+    # for the inverse survival function tests, swap ref and x
+
+    @pytest.mark.parametrize('x, c, ref',
+                             [(1e20, 0.1, 1.8600379880103705e-44),
+                              (1e5, 0.5, 2.306726997904701e-138)])
+    def test_sf_isf(self, x, c, ref):
+        assert_allclose(stats.dweibull.sf(x, c), ref, rtol=5e-14)
+        assert_allclose(stats.dweibull.isf(ref, c), x, rtol=5e-14)
```

Comment:
Let's convert this to a test for `weibull_min.sf`/`isf`. The test for `dweibull.sf`/`isf` can follow the style of `test_entropy`. There the comment doesn't need to be repeated, though.